OOP/OOPS_Session_1/Matrix.java

```diff
@@ -0,0 +1,61 @@
+import java.util.*;
+public class Matrix 
+{
+	int data[][];
+	int noRows;
+	int noCols;
+	private Matrix Matrix;
+	//Matrix class constructor used to initialize variables
+	public Matrix(int row,int col)
+	{
+		this.noRows=row;
+		this.noCols=col;
+		data=new int[row][col];
+	}
+	//addElement method used to add elements in matrix
+	public void addElement(int row,int col,int val)
+	{
+	    data[row][col]=val;
+	}
+	//This method is used to transpose the Matrix
+	public Matrix Transpose()
```

Comment:
ACTION: Use proper convention for method name , Method and variable names should start with small letter.